src/redux/rockets/reducer.js

```diff
@@ -14,6 +15,12 @@ const rocketsReducer = (state = [], action) => {
         return { ...rocket, reserved: !rocket.reserved };
       });
 
+    case CANCEL_ROCKET:
+      return state.map((rocket) => {
+        if (rocket.id !== action.payload) return rocket;
+        return { ...rocket, reserved: false };
+      });
```

Comment:
Well, when almost the entire concept of the reserveRocket and the cancelRocket is the same, what do you think to remove the one which smells duplicated?